spec/ask_export/report_spec.rb

```diff
@@ -4,17 +4,36 @@
   end
 
   describe "#since_time" do
-    it "returns 10am on the previous day in current time zone" do
+    it "defaults to 10am on the previous day in the current time zone" do
       expect(described_class.new.since_time)
         .to eq(Time.zone.parse("2020-04-30 10:00"))
     end
+
+    it "can be overridden with the SINCE_TIME environment variable" do
```

Comment:
Can we also add tests for the relative scenarios of these variables?

Reply:
Sure, I've added one in. I felt like it was covered with the default since it exercises the same code but you're right it's better to have an explicit one.

Reply:
Ah yes, that's true. I noticed this in the context of the commit, which didn't need to change the integration tests, which will indeed be testing this implicitly.

Reply:
Or maybe not - we're setting it quite explicitly in a few tests. Anyway, should we also have one for the `UNTIL_TIME`?

Reply:
I didn't give as rich a collection of tests for until time as they just test the same functionality of code (in the private `parse_time` method) and just focused on what was specifically different.

I've come around more to the perspective that unit tests are coupled to the implementation and so that it'd only be a concern to test these if we dropped the `#since_time` method. I appreciate though that it leaves a bit of "why do we care there but not here?" but I think that is pretty understandable by looking at the code.